### ISSUE TITLE
Complete theme for `catppuccin`

### ISSUE THREAD

**Issue by d4ilyrun** [d4ilyrun/nixpkgs#6]
## Summary

Looking into [catppuccin](https://github.com/catppuccin), there are multiple packages that applications that include an official catppuccin theme.

After #2 has been merged, it is easily possible to complete our theme, by combining derivations, `home-manager` configurations and those repositories.

## TODO

Here's a list of what's available that could be used inside our configuration. Some of these may not be reproducible using `Nix`, or are already done (but could be reviewed).

- [ ] GTK
- [ ] thunderbird
- [ ] Wallpapers
- [ ] Discord
- [ ] Refind
- [ ] spicetify
- [ ] rofi
- [ ] Shell
	- [ ] Tmux
	- [ ] bat
	- [ ] starship
- [ ] firefox
	- [ ] chrome.css
	- [ ] dark-reader
- [ ] dunst
